types/logger.d.ts

```diff
@@ -61,4 +61,5 @@ export interface FastifyLoggerOptions<
   };
   level?: string;
   genReqId?: <RequestGeneric extends RequestGenericInterface = RequestGenericInterface>(req: FastifyRequest<RequestGeneric, RawServer, RawRequest>) => string;
+  prettyPrint?: boolean;
```

Comment:
This is incomplete as the definition can either be a `boolean` or a `pino-pretty` option object. Please see: https://github.com/pinojs/pino-pretty#options

Reply:
Good point.  I will update.